addons/website_event_track/controllers/event_track.py

```diff
@@ -149,13 +149,20 @@ def _event_tracks_get_values(self, event, tag=None, **searches):
         if tracks_announced:
             tracks_announced = tracks_announced.sorted('wishlisted_by_default', reverse=True)
             tracks_by_day.append({'date': False, 'name': _('Coming soon'), 'tracks': tracks_announced})
+        # Check if there are any ongoing or upcoming tracks
+        has_upcoming_or_ongoing = any(track for track in tracks_sudo if not track.is_track_done)
 
         for tracks_group in tracks_by_day:
-            # the tracks group is folded if all tracks are done (and if it's not "today")
-            tracks_group['default_collapsed'] = (today_tz != tracks_group['date']) and all(
-                track.is_track_done and not track.is_track_live
-                for track in tracks_group['tracks']
-            )
+            # If there are upcoming or ongoing tracks
+            if has_upcoming_or_ongoing:
+                # Fold past days where all tracks are done
+                if tracks_group['date'] and all(track.is_track_done for track in tracks_group['tracks']):
+                    tracks_group['default_collapsed'] = True
+                else:
+                    tracks_group['default_collapsed'] = False
+            else:
+                # If all tracks are done, expand all the tracks
+                tracks_group['default_collapsed'] = False
```

Comment:
No need to use an if-else here, you can directly assign `tracks_group['default_collapsed']`
```suggestion
                tracks_group['default_collapsed'] = (
                    has_upcoming_or_ongoing and
                    tracks_group['date'] and all(track.is_track_done for track in tracks_group['tracks'])
                )
```